src/bsd.c

```diff
@@ -926,6 +932,77 @@ main(int argc, char **argv)
 }
 #endif /* BOOLEXP_DEBUGGING */
 
+void load_plugins() {
+  typedef int plugin_init();
+
+  DIR* pluginsDir;
+  struct dirent* in_file;
+  void* testPlugin;
+  char plugin_name[256];
+
+  plugin_head = NULL;
+  plugin_last = NULL;
+  plugin_curr = NULL;
+
+  if (NULL != (pluginsDir = opendir("../plugins"))) {
+    while ((in_file = readdir(pluginsDir)))
+    {
+      if (!strcmp(in_file->d_name, ".")) continue;
+      if (!strcmp(in_file->d_name, "..")) continue;
+      if (!strstr(in_file->d_name, ".so")) continue;
+
+      memset(plugin_name, 0, strlen(plugin_name));
+      snprintf(plugin_name, sizeof(plugin_name), "../plugins/%s", in_file->d_name);
+
+      do_rawlog(LT_ERR, "Found: %s ", plugin_name);
```

Comment:
This message is too generic to be helpful and should include the word 'plugin' somewhere.